modules/nw-anp-np-reference.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/logging-network-security.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-anp-differences-networkpolicy_{context}"]
+= Key differences between AdminNetworkPolicy and NetworkPolicy custom resources
+
+[cols="1,1,1"]
+|===
+|Policy elements | AdminNetworkPolicy | NetworkPolicy
+
+|Applicable user
+|Cluster administrator or equivalent
+|Namespace owners
+
+|Scope
+|Cluster
+|Namespaced
+
+|Drop traffic
+|Supported with an explicit `Deny` action set as in a rule.
+|Supported via implicit `Deny` isolation at policy creation time.
+
+|Delegate traffic
+|Supported with an `Pass` action set as a rule.
+|Not applicable
+
+|Allow traffic
+|Supported with an explicit `Allow` action set via a rule.
```

Comment:
You have three different ways of phrasing here - "set as in a rule", "set as a rule" and "set via a rule". I don't have a strong feeling but whichever you choose, I'd standardize them